pages/docs/widgets/variants/button.en.mdx

```diff
@@ -10,29 +10,39 @@ import { RequestToken } from '@/components/intermediary/request-token'
 
 This widget renders only a button, which when clicked, opens our calculator.
 
-It doesn’t required any equipment data to work, so you can place this button anywhere on your website.    
+It doesn’t required any equipment data to work, so you can place this button anywhere on your website.  
 The widget has several configuration parameters, which you can read about below.
 
 ## Getting started
 
 Add a piece of HTML to your web page with class `beequip-button`, in a location where you want the button to render.
 
 ```html
-<a href="https://lease.beequip.nl" class="beequip-button">Lease equipment bij Beequip</a>
+<a href="https://lease.beequip.nl" class="beequip-button"
+    >Lease equipment bij Beequip</a
+>
```

Comment:
This is weird formatting of HTML. I've never seen this before. I would expect either this:

```html
<a href="https://lease.beequip.nl" class="beequip-button">Lease equipment bij Beequip</a>
```

or this

```html
<a 
   href="https://lease.beequip.nl" 
   class="beequip-button"
>
   Lease equipment bij Beequip
</a>
```

 So I would suggest to revert back to the original.

Reply:
Manually reverted the changes